spec/unit/facter/sudo_version_spec.rb

```diff
@@ -1,57 +0,0 @@
-require 'spec_helper'
-require 'facter/sudo_version'
-
-describe Facter::Util::SudoVersion do
-
-  with_fixtures = {
```

Comment:
How come the unit tests went away?

Reply:
Copied the fact from the (internal) sudoers module and it had no unit tests available. Didn't want to put time to create them as it seems that the sudo module usage will be deprecated sooner than later.